go.mod

```diff
@@ -86,16 +89,18 @@ require (
 	gopkg.in/go-playground/validator.v9 v9.30.2
 	gopkg.in/natefinch/lumberjack.v2 v2.2.1
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.28.9
-	k8s.io/apiextensions-apiserver v0.28.9
-	k8s.io/apimachinery v0.28.9
-	k8s.io/apiserver v0.28.9
-	k8s.io/client-go v0.28.9
+	gopkg.in/yaml.v3 v3.0.1
+	helm.sh/helm/v3 v3.15.3
+	k8s.io/api v0.30.0
+	k8s.io/apiextensions-apiserver v0.30.0
+	k8s.io/apimachinery v0.30.0
+	k8s.io/apiserver v0.30.0
+	k8s.io/client-go v0.30.0
 	k8s.io/code-generator v0.28.9
-	k8s.io/component-base v0.28.9
+	k8s.io/component-base v0.30.0
 	k8s.io/klog/v2 v2.120.1
 	k8s.io/kube-aggregator v0.28.9
-	k8s.io/kube-openapi v0.0.0-20230717233707-2695361300d9
+	k8s.io/kube-openapi v0.0.0-20240228011516-70dd3763d340
```

Comment:
These are too new. It will fail production build. We still have an on-going PR for [k8s v1.29](https://github.com/projectcalico/calico/pull/8805).

Reply:
Will resolve this.